dc-cudami-admin/dc-cudami-admin-webapp/src/main/java/de/digitalcollections/cudami/admin/controller/identifiable/resource/FileResourcesMetadataController.java

```diff
@@ -118,6 +128,24 @@ public ResponseEntity save(@RequestBody FileResource fileResource)
     }
   }
 
+  @GetMapping("/api/fileresources/images")
+  @ResponseBody
+  public SearchPageResponse<FileResource> search(
```

Comment:
Der Name sollte kennzeichnen dass es hier um Images geht und nicht um File Resources generell.

Reply:
Werde ich in `searchImages` umbenennen.